File: RateIt.Common/Core/QueryResults/UserLoginQueryResult.cs
using System;

namespace RateIt.Common.Core.QueryResults
{
    [Serializable]
    public class UserLoginQueryResult : BaseQueryResult
    {

#region Properties

        public string UserId;
        public string SessionId;

#endregion

#region Class methods

    public UserLoginQueryResult() { }

    public UserLoginQueryResult(byte[] userId, byte[] sessionId)
    {
        UserId = Convert.ToBase64String(userId);
        SessionId = Convert.ToBase64String(sessionId);
    }

#endregion

    }
}

File: RateItAndroid/src/com/RateIt/entity/UserLoginQueryResult.java
package com.RateIt.entity;

/**
 * Created by igor on 7/16/14.
 */
public class UserLoginQueryResult extends BaseQueryResult {

    public String UserId;
    public String SessionId;

    public String getUserId() {
        return UserId;
    }

    public void setUserId(String userId) {
        UserId = userId;
    }

    public String getSessionId() {
        return SessionId;
    }

    public void setSessionId(String sessionId) {
        SessionId = sessionId;
    }
}
